pkg/database/fake/database_generated.go

```diff
@@ -47,54 +47,54 @@ type LogUrlRow struct {
 	JsonPath      string
 }
 
-type Database struct {
+type fakeDatabase struct {
```

Comment:
I don't know if the `fakeDatabase` type should be unexported. I know the sink uses this type for some it's unit tests

Reply:
indeed, but I think there is no problem of having unexported if we have the "initializer" exported.

As I see it, all that needed to be exported related to database from API and sink POV are:
* The different constructors (`NewDatabase`, `NewFakeDatabase`)
* The interface `DBInterface` (or `DBReader` and `DBWriter`)